AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -61,6 +61,8 @@ rasp.yandex.*##.TravelHotelsBanner
 @@||yastatic.net/daas/atom.js$domain=mail.yandex.by|mail.yandex.kz|mail.yandex.ru|mail.yandex.com|mail.yandex.com.tr
 !-----------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/134866
+||cdn.tinypass.com/api/tinypass.min.js$domain=digiday.com
```

Comment:
Usually we do not block notify about limited number of free articles, but the counter is blocked here, so can be merged.